doc/howto/cluster_recover.md

```diff
@@ -6,43 +6,38 @@ In that case, no operations are possible on this member, and neither are operati
 See {ref}`clustering-offline-members` and {ref}`cluster-automatic-evacuation` for more information.
 
 If you can bring the offline cluster members back or delete them from the cluster, operation resumes as normal.
-If this is not possible, there are a few ways to recover the cluster, depending on the scenario that caused the failure.
-See the following sections for details.
+If the cluster members are lost permanently (e.g. disk failure), it is possible to recover them if at least one member survives.
 
 ```{note}
-When your cluster is in a state that needs recovery, most `lxc` commands do not work, because the LXD client cannot connect to the LXD daemon.
+When your cluster is in a state that needs recovery, most `lxc` commands do not work, because the LXD daemon cannot connect to the LXD database.
```

Comment:
`lxc` is the LXD client command, `lxd` is the LXD daemon command - what I was trying to explain here is why `lxc` commands don't work and that you have to use `lxd` instead.
Maybe just keep "LXD client"? But I'm not sure of the technical details. The way it's written now is confusing though imo.

Reply:
My understanding is that the client has no trouble connecting to the daemon; when the daemon then makes a request to dqlite, dqlite never returns a reply. That blocks the client request.

How about "When your cluster is in a state that needs recovery, most `lxc` commands do not work, because the LXD database does not respond when a majority of database voters are inaccessible"?

The choice to use `lxd` is likely because `lxc` is only a client; it should interact with LXD via a socket/TCP conn instead of directly modifying database files. These two things should probably be in separate notes. I'll rework this.

Reply:
Yes, that sounds better. The important part is to point out that we're using different commands here (before this part was added, we go feedback that `lxc cluster` doesn't have any of those mentioned subcommands).